client/lib/plans/features-list.js

```diff
@@ -547,15 +547,15 @@ export const FEATURES_LIST = {
 
 	[ constants.FEATURE_3GB_STORAGE ]: {
 		getSlug: () => constants.FEATURE_3GB_STORAGE,
-		getTitle: () => i18n.translate( '3GB Storage Space' ),
+		getTitle: () => i18n.translate( '3 GB Storage Space' ),
 		getDescription: () =>
 			i18n.translate( 'Storage space for adding images and documents to your website.' ),
 	},
 
 	[ constants.FEATURE_6GB_STORAGE ]: {
 		getSlug: () => constants.FEATURE_6GB_STORAGE,
 		getTitle: () =>
-			i18n.translate( '{{strong}}6GB{{/strong}} Storage Space', {
+			i18n.translate( '{{strong}}6 GB{{/strong}} Storage Space', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **25** times:
`translate( '{{strong}}200 GB{{/strong}} Storage Space' )` *ES Score: 7*